### ISSUE TITLE
Death State not working, custom Estimation Settings not applying.

### ISSUE THREAD

**Issue by maddoodles** [mclemente/healthEstimate#169]
I'm running Wrath & Glory 4.2.0 in Foundry 10, build 29.

Death States do not apply to tokens when the Death Condition is applied, I have pointed to the skull icon that system uses, plus other icons to test, and it does not change from the Unconscious state.

I also cannot set custom Estimations without changing the defaults, I'm not sure how custom tables are supposed to be used so I don't know if this is a bug or if there's a setting to apply them I'm missing.

Thank you for your time.

**Comment by mclemente** [comment]
Death Condition is bugged for Wrath & Glory, I'll work on a fix.

As for the custom Estimations, what are you setting as the JS Rule?

I've also added a small wiki to give some explanation what the settings do https://github.com/mclemente/healthEstimate/wiki/Estimation-Settings

**Comment by maddoodles** [comment]
I've currently not put anything there, I didn't know if anything did need to be there and couldn't find instruction on that at the time, I see what I need to do now, I think.

**Comment by mclemente** [comment]
I can't help you if you don't tell me what you want to do.

**Comment by maddoodles** [comment]
> I also cannot set custom Estimations without changing the defaults, I'm not sure how custom tables are supposed to be used so I don't know if this is a bug or if there's a setting to apply them I'm missing.

I intended to change the health states but couldn't get any tokens to use them, I had to overwrite the defaults which I got the impression wasn't the intended way to do it anymore. I see the wiki has expanded a lot since however, that doesn't seem to be how they're intended to be used.

Anyway, thank you for addressing W&G compatibility, it's much appreciated.
